curriculum/challenges/english/20-upcoming-python/learn-regular-expressions-by-building-a-password-generator/656489e0a5d0954c1a64fc93.md

```diff
@@ -0,0 +1,37 @@
+---
+id: 656489e0a5d0954c1a64fc93
+title: Step 7
+challengeType: 20
+dashedName: step-7
+---
+
+# --description--
+
+Now print `all_characters` to see what it looks like.
+
+# --hints--
+
+You should print your `all_characters` variable.
+
+```js
+({ test: () => assert.match(e.code.original["main.py"], /^print\s*\(\s*all_characters\s*\)/m) })
+```
+
+# --seed--
+
+## --seed-contents--
+
+```py
+--fcc-editable-region--
+import string
+
+
+# Define the possible characters for the password
+letters = string.ascii_letters
+digits = string.digits
+symbols = string.punctuation
+
+# Combine all characters
+all_characters = letters + digits + symbols
+--fcc-editable-region--
+```
```

Comment:
Editable region can be shortened but it is also interfering with the tests.
```suggestion
```py
import string


# Define the possible characters for the password
letters = string.ascii_letters
digits = string.digits
symbols = string.punctuation
--fcc-editable-region--
# Combine all characters
all_characters = letters + digits + symbols
--fcc-editable-region--
```
```

Reply:
What do you mean with "interfering with the tests"? I don't see any problem with this step.